README.md

```diff
@@ -13,6 +13,37 @@
 * (optional) Change server ip/port number in config.
 * All other packages that generate error "No module named" does exist, must be installed.
 
+## Installation and Setup 
+
+Clone the repository and move into the directory 
+
+```
+git clone https://github.com/oduwsdl/CarbonDate
+cd CarbonDate 
+```
+
+Install python requirements 
+
+```
+pip install -r requirements.txt 
+```  
+
+### Missing Aysnchronous attribute 
+
+![Example Error](https://github.com/njenn001/CarbonDate/blob/master/images/Error.jpg)
```

Comment:
I'm kind of curious where this error stems from. I don't have issues with the docker hub image.

Maybe there is a Python version and library version mismatch going on? The docker image is python 3.7 with the following library versions:
```
$ docker run -it --rm oduwsdl/carbondate bash
root@4d968fac6b3c:/usr/src/app# pip3 list --format freeze
atomicwrites==1.1.5
attrs==18.1.0
beautifulsoup4==4.6.3
certifi==2018.8.13
chardet==3.0.4
idna==2.7
lxml==4.2.4
more-itertools==4.3.0
pip==18.0
pluggy==0.7.1
py==1.5.4
pytest==3.7.1
python-dateutil==2.7.3
requests==2.19.1
requests-file==1.4.3
setuptools==40.0.0
six==1.11.0
surt==0.3.0
tldextract==2.2.0
tornado==5.1
urllib3==1.23
wheel==0.31.1
```